ds2799/model-ds2799-desc-compcop_smdl.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Name": "ds2799_compcor",
+  "Description": "scrubbing compcor, see https://nilearn.github.io/stable/modules/generated/nilearn.interfaces.fmriprep.load_confounds_strategy.html",
+  "BIDSModelVersion": "1.0.0",
+  "Input": {
+    "task": [
+      "rest"
+    ],
+    "space": [
+      "MNI152NLin2009cAsym"
```

Comment:
Since there's no combination across subjects, it's not clear that we need to define the space.

```suggestion
```